binzx/installer

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+readonly script=otomi
+readonly branch=otomi-zx
+readonly remote=https://raw.githubusercontent.com/redkubes/otomi-core/$branch/binzx/$script
+readonly paths=(
+    /usr/local/bin
+    /usr/bin
```

Comment:
why could otomi be found in /usr/bin or /bin? We don't have code that ever put it there...is this one of those defenses against impossible use cases? Why should you check in an array of paths? I would presume the default location would be just `./bin` as that does not involve the user's approval...if the user wants to put it somewhere else that is entirely up to them (and should be mentioned in the docs as such).